gradle/dokka.gradle

```diff
@@ -0,0 +1,72 @@
+/*
```

Comment:
If it's not so much effort, maybe we could translate this file to kts :D

Reply:
@yamilmedina I have no clue what to do here, so I leave it to you folks to fix this one